app/github_public_repo/README.md

```diff
@@ -0,0 +1,20 @@
+
+# SARS-CoV-2 Serosurveillance Data by Serotracker
+
+[SeroTracker](https://serotracker.com/en/Explore) synthesizes findings from thousands of COVID-19 seroprevalence studies worldwide, providing a data platform and interactive dashboard for SARS-CoV-2 serosurveillance. This dataset in this repository represents our collection of serosurveillance studies.
+
+### Download
+Our complete SARS-Cov-2 dataset can be scraped programmatically from this repository in the following ways:
+1. Using the command: `curl https://raw.githubusercontent.com/serotracker/sars-cov-2-data/main/test.csv`.
+2. Using a GET request through REST API protocol.
+
+Our dataset can also be downloaded directly from our [Airtable view]([https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on](https://airtable.com/shraXWPJ9Yu7ybowM/tbljN2mhRVfSlZv2d?backgroundColor=blue&viewControls=on)).
+
+The CSV file follows the format of 1 row per serosurveillance study.
+ 
+### Additional Information
+Please see our [Data Dictionary]([https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0](https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit#gid=0)) for explanations of our variables, data types, and descriptions as well as insight into how our data is collected by our research team.
+
+In order to keep up to date with important changes to our dataset, please consult our [Change Log](https://airtable.com/shrxpAlF6v0LeRYkA/tblC6jj904WXUzwVY) regularly.
+
+For more information about how we collect, extract and use our data, please see the [Data page]([https://serotracker.com/en/Data](https://serotracker.com/en/Data)) on our website.
```

Comment:
Some suggestions of things to add (some based on OWIDs readme):

- Information on how to cite us, licensing if necessary, etc
- Form for submitting sources
- Form for people who want to contact us (similar to data download, will give us a chance to collect contact info about people who are down to use our data)

@simonarocco @annoyingpr 

cc @rahularoradfs

Reply:
made a [new form](https://forms.gle/gqi3kvKQgasYCrQE9) for this, and pushed updates based on your suggestions!